source/configuration_manual/dovecot_ssl_configuration.rst

```diff
@@ -223,13 +207,11 @@ By default Dovecot's allowed ciphers list contains:
 
 Disallowing more won't really gain any security for those using better ciphers, but it does prevent people from accidentally using insecure ciphers. See https://www.openssl.org/docs/manmaster/man1/ciphers.html for a list of the ciphers.
 
-You should usually prefer server ciphers and their order, so setting
-
 .. code::
 
-  ssl_prefer_server_ciphers = yes
+  ssl_prefer_server_ciphers = no
 
-is recommended.
+For TLSv1.3 server ciphers should not longer be preferred.
```

Comment:
I'd move this sentence above the ssl_prefer_server_ciphers and have it end with `:`.